src/main/java/com/github/tomakehurst/wiremock/stubbing/ServeEvent.java

```diff
@@ -46,7 +46,7 @@ public class ServeEvent {
   private final LoggedResponse response;
   private final Timing timing;
 
-  private final ConcurrentLinkedQueue<SubEvent> subEvents;
+  private final ConcurrentLinkedDeque<SubEvent> subEvents;
```

Comment:
I don't think we should change it here since this collection is never likely to get beyond single digits in length and we never reverse it.

Also, this is a breaking change, and while we can still do this it's got me thinking that maybe we should expose this as a List on the public methods of this class rather than a Queue.

Reply:
Done